virtual-setup/slack-procedures.md

```diff
@@ -1,13 +1,12 @@
 ---
 title: Using Slack for CCDL virtual workshops
 ---
-We use Slack for communication during CCDL virtual workshops.
 
-<p><img style = "padding: 0 15px; float: left;" src = "screenshots/slack-cancer-data-science-logo.png" width = "75"></p>
-<p style="margin-top: 20px;"> </p>
-<p>Specifically, we use the <b>Cancer Data Science Slack</b> team administered by the CCDL.
-You can join Cancer Data Science Slack by following this link: <a href="https://www.ccdatalab.org/slack">ccdatalab.org/slack</a></p>
-<br>
+
+<img style = "padding: 0px 15px 0px 0px; float: left;" src = "screenshots/slack-cancer-data-science-logo.png" width = "75">
+Specifically, we use the **Cancer Data Science Slack** team administered by the CCDL.
```

Comment:
Should we do some of this now or wait do a whole re-brand all at once? Up to you.
```suggestion
Specifically, we use the **Cancer Data Science Slack** team administered by the Data Lab.
```